pyboleto/bank_api/itau.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
+from pyboleto.bank.itau import BoletoItau
+
+import json
+import requests
+
+
+def limpa_formatacao(cnpj_cpf):
+    '''
+    Limpa os caracteres de formatação
+    :param cnpj_cpf: CPF ou CNPJ de entrada
+    :return: Retorna o CPF ou CNPJ sem formatação
+    '''
+    return cnpj_cpf.replace('.', '').replace('-', '').replace('/', '').replace(
```

Comment:
Talvez seja mais prudente e mais rápido se fossem mantidos apenas os dígitos, ao invés de remover apenas caracteres específicos. Por exemplo: `return ''.join([c for c in cnpj_cpf if c.isdigit()])`.